functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,150 @@
+# inflate-helm-chart
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `inflate-helm-chart` function inflates a local or remote Helm chart. 
+
+<!--mdtogo-->
+
+You can learn more about helm charts [here][helm].
+
+## Usage
+
+This function can be used with any KRM function orchestrators (e.g. kpt or kustomize) 
+to inflate a specified Helm chart.
+It can be used imperatively or declaratively when run with kustomize. With kpt, it can be
+run imperatively or declaratively as an exec function, or just imperatively as a container
+function. 
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+There are 2 kinds of `functionConfig` supported by this function:
+
+- `ConfigMap`
+- A custom resource of kind `HelmChartInflator`
+
+To use a `ConfigMap` as the `functionConfig`, the desired parameters must be
+specified in the `data` field:
+
+```yaml
+data:
+  chartHome: string
+  configHome: string
+  name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesInline: map[string]interface{}
+  valuesFile: string
+  valuesMerge: string
+  includeCRDs: bool
+```
+
+A `functionConfig` of kind `HelmChartInflator` has the following supported parameters: 
+
+```yaml
+helmGlobals:
+  chartHome: string
+  configHome: string
+helmCharts:
+- name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesInline: map[string]interface{}
+  valuesFile: string
+  valuesMerge: string
+  includeCRDs: bool
+```
+
+#### Parameter Descriptions
```

Comment:
I'd suggest you to merge this section with the "schema" above, since the 2 formats of functionConfig are not the same. The CRD one has more fields and the fields can be more than string.

Reply:
I merged it to clarify the differences between the two functionConfigs